Server/middleware/auth.js

```diff
@@ -1,4 +1,4 @@
-async function authenticate(req, res, next) {
+const authenticate = async (req, res, next) => {
   const authInfo = await getUserAuthInfo(req);
```

Comment:
need to require getUserAuthInfo from utils.js (didn't notice it in my previous changes, my bad)